@vates/disposable/USAGE.md

```diff
@@ -1,13 +1,19 @@
+This examples use the library [promise-toolbox/resource-management](https://github.com/JsCommunity/promise-toolbox#resource-management)
+
 ```js
```

Comment:
Add some documentation like the following:

```suggestion
### `deduped(fn, keyFn)`

Creates a new function that wraps `fn` and instead of creating new disposables at each call, returns copies of the same one when `keyFn` returns the same keys.

Those copies contains the same value and can be disposed independently, the source disposable will only be disposed when all copies are disposed.

`keyFn` is called with the same context and arguments as the wrapping function and must returns an array of keys which will be used to identify which disposables should be grouped together.

```js
```